pom.xml

```diff
@@ -199,6 +199,23 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>knative</id>
+            <activation>
+                <property>
+                    <name>knative</name>
+                </property>
+            </activation>
+            <properties>
+                <quarkus.container-image.image>${env.KO_DOCKER_REPO}/${project.artifactId}:${env.TAG}</quarkus.container-image.image>
```

Comment:
@christophd @matzew currently the module names by directory and artifactId are different. E.g. `  <artifactId>kn-connector-aws-s3-source</artifactId>` for aws-s3-sink. Should we align those, or create a name property to be reflected in the image name.

Reply:
I see Eventing core is expecting kn prefixed name. That's fine for now.

```
if source.Spec.Aws.S3 != nil {
			return "quay.io/openshift-knative/kn-connector-aws-s3-source:1.0-SNAPSHOT"
		}
```

Reply:
:see_no_evil: 

let's fix that :rofl: